src/Features/Core/Portable/Diagnostics/Analyzers/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -132,6 +123,6 @@ private bool IsFrameworkTypePreferred(OptionSet optionSet, PerLanguageOption<Cod
         /// </summary>
         /// <remarks>if predefined type is not preferred, it implies the preference is framework type.</remarks>
         private static bool OptionSettingPrefersFrameworkType(CodeStyleOption<bool> optionValue, ReportDiagnostic severity)
-            => !optionValue.Value && severity.WithDefaultSeverity(DiagnosticSeverity.Hidden) < ReportDiagnostic.Hidden;
```

Comment:
is it using supress as its own special meaning now?

Reply:
`Suppress` is the internal representation of severity `none` from .editorconfig. Unlike rule set files, we cannot blanket disable an analyzer or diagnostic with severity `none` so we need a way to represent it in the options passed to the analyzers.